### PR TITLE
panels now receive their size and not the one of the whole window

### PR DESCRIPTION
**Pull request by manuel3108**

### CODE REVIEW

**Comment by manuel3108** [comment]
please do not review atm. I found another problem, which should be fixed first

File: src/main/java/org/gleisbelegung/ui/lib/panel/Panel.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
  */
 public abstract class Panel implements PanelInterface{
     private boolean visibility = true;
+    private boolean isFirstResize = true;
```
Comment (Greonyral) [inline src/main/java/org/gleisbelegung/ui/lib/panel/Panel.java]: private variable never read in the class => looks pretty useless.

File: src/main/java/org/gleisbelegung/ui/lib/panel/Panel.java [hunk 2]
```diff
@@ -22,4 +23,13 @@ public void setVisible(boolean visibility){
...
     }
+
+    @Override
+    public void onResize(double width, double height) {
+        onResize(width, height, isFirstResize);
+        isFirstResize = false;
+    }
+
+
+    public abstract void onResize(double width, double height, boolean isFirstResize);
```
Comment (Greonyral) [inline src/main/java/org/gleisbelegung/ui/lib/panel/Panel.java]: the parameter `isFirstResize` seems to be never used

Comment (manuel3108) [reply src/main/java/org/gleisbelegung/ui/lib/panel/Panel.java]: Uups, I should have written a comment to explain this.

In this PR they are not used but in #4. Especially this commit 747f9acdce1ce34203d053b1e9d675b613d057d3. This branch is already merged in #4, even withoud beeing approved.

Same applies for [#10 Comment 1](https://github.com/manuel3108/Gleisbelegung/pull/10#discussion_r306398676)

**Pull request merged**